e2e-tests/gef/cypress/e2e/clone/facility-end-date-enabled/clone-v0-to-v1.spec.js

```diff
@@ -0,0 +1,86 @@
+import relative from '../../relativeURL';
+import { BANK1_MAKER1 } from '../../../../../e2e-fixtures/portal-users.fixture';
+import cloneGEFDeal from '../../pages/clone-deal';
+import mandatoryCriteria from '../../pages/mandatory-criteria';
+import nameApplication from '../../pages/name-application';
+import applicationDetails from '../../pages/application-details';
+import facilities from '../../pages/facilities';
+import { submitButton } from '../../partials';
+import aboutFacilityUnissued from '../../pages/unissued-facilities-about-facility';
+
+/**
+ * NOTE: These tests check the backwards compatibility with in-flight version 0 deals.
+ * A migration may need to be run on production if this test is updated.
+ */
+context('Clone version 0 deal to version 1', () => {
+  let version0DealId;
+  let clonedDealId;
+
+  before(() => {
+    cy.insertVersion0Deal(BANK1_MAKER1.username).then(({ insertedId: dealId }) => {
+      version0DealId = dealId;
+    });
+  });
+
+  beforeEach(() => {
+    cy.saveSession();
+  });
+
+  describe('cloning a version 0 deal', () => {
+    before(() => {
+      cy.login(BANK1_MAKER1);
+      cy.visit(relative(`/gef/application-details/${version0DealId}`));
+
+      applicationDetails.addContingentFacilityButton().click();
+      facilities.hasBeenIssuedRadioYesRadioButton().click();
+      cy.clickContinueButton();
+
+      cy.visit(relative(`/gef/application-details/${version0DealId}`));
+
+      cloneGEFDeal.cloneGefDealLink().click();
+      mandatoryCriteria.trueRadio().click();
+
+      cy.clickContinueButton();
```

Comment:
The `clone-gef-deal.spec` tests use `form().submit();` rather than these commands - @ttbarnes should these be replaced everywhere? I dont really like that this manipulates the `form` element directly rather than clicking a button on screen

Reply:
Wowsa, didn't realise, sorry I didn't spot this before.

I completely agree, can we replace all instaces of `form().submit()` please, there are 25x instances in 4x files

Reply:
Have made a ticket https://ukef-dtfs.atlassian.net/browse/DTFS2-7589 but I probably wont have time this sprint to take a look at this.